clients/elrond/client.go

```diff
@@ -317,6 +333,24 @@ func (c *client) PerformAction(ctx context.Context, actionID uint64, batch *clie
 	return hash, err
 }
 
+func (c *client) CheckClientAvailability(ctx context.Context) error {
```

Comment:
missing comment + make this concurrent safe

Reply:
done